tests/Unit/Admin/Core/Extension/PublishTimePeriodExtensionTest.php

```diff
@@ -1,39 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony CMF package.
- *
- * (c) 2011-2015 Symfony CMF
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\Tests\Unit\Admin\Core\Extension;
-
-use Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\Admin\Core\Extension\PublishTimePeriodExtension;
-
-class PublishTimePeriodExtensionTest extends \PHPUnit_Framework_TestCase
```

Comment:
They literally test nothing and were harder to write as there are now chained tab with calls, etc.